UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FulfilmentDataService.cs

```diff
@@ -214,6 +216,28 @@ private async Task<(string, List<FssBatchFile>)> DownloadEssExchangeSet(string e
             return (downloadPath, files);
         }
 
+        private List<FssBatchFile> RenameFiles(string downloadPath, List<FssBatchFile> files, Batch batchType)
+        {
+            foreach (var file in files)
+            {
+                IFileInfo fileInfo = _fileSystemHelper.GetFileInfo(Path.Combine(downloadPath, file.FileName));
+                string weekNumber = CommonHelper.GetCurrentWeekNumber(DateTime.UtcNow).ToString();
+                string currentYear = DateTime.UtcNow.ToString("yy");
+
+                if (batchType == Batch.EssFullAvcsZipBatch)
+                {
+                    int mediaNumber = int.Parse(file.FileName.Substring(1, 2));
```

Comment:
Can we rename it to dvdNumber same as funtional test to make it consistent.
```suggestion
                    int dvdNumber = int.Parse(file.FileName.Substring(1, 2));
```

Reply:
Changed.